index.html

```diff
@@ -10,7 +13,17 @@
     <script>
       let ui = null
       window.onload = () => {
-        TidyBlocksUI = tidyblocks.setup('en', 'root')
+        // Remove the Jekyll front matter if it's there. Used to develop locally
+        // without running Jekyll.
+        if (document.body.childNodes[0].data.includes('layout: null')){
```

Comment:
Why do you prefer this to looking for a language in the URL `https://tidyblocks.tech/en/`?

Reply:
So the problem is pointing the different language URLs to index.html (i.e. how does it know https://tidyblocks.tech/en/ and https://tidyblocks.tech/es/ point to index.html).

If there's a way to do that without needing Jekyll involved with `index.html` would definitely be in favour.